ci/jenkins/Jenkinsfile_unix_cpu

```diff
@@ -37,17 +37,17 @@ core_logic: {
     custom_steps.compile_unix_cpu_openblas('cpu'),
     custom_steps.compile_unix_openblas_debug_cpu('cpu_debug'),
     custom_steps.compile_unix_mkl_cpu('cpu_mkl'),
-    custom_steps.compile_unix_mkldnn_cpu('mkldnn_cpu'),
-    custom_steps.compile_unix_mkldnn_mkl_cpu('mkldnn_mkl_cpu'),
+    custom_steps.compile_unix_onednn_cpu('mkldnn_cpu'),
```

Comment:
for me it seems that "lib_name" parameter also could be easily changed to onednn_cpu

Reply:
Yes. The only requirement is to be consistent. (Ie. use the same name for the compile as well as the test job)

Reply:
It is already listed in proposed changes under "Change names of created in Jenkins files instances of mxnet using onednn".